fileCoverage.ts

```diff
@@ -28,24 +31,28 @@ export class FileCoverage {
   }
 
   get missingCoverage(): string {
-    const contig: Range[] = [
+    const ranges: Range[] = [
       new Range(),
     ];
 
-    for (let i = 0; i < this.untouchedLines.length; i++) {
-      const recent = contig.at(-1) as Range;
-      if (recent.from === -Infinity) {
-        recent.from = recent.to = this.untouchedLines[i];
-      } else if (this.untouchedLines[i] === recent.to + 1) {
-        recent.to = this.untouchedLines[i];
+    const orderedLines = Array
+      .from(this.lines.entries())
```

Comment:
Believe Deno > 1.31 supports the `.toSorted` method, which we could use here if we preferred. AFAIK there's no reason outside of coding style to use it, just wanted to purvey that info.